samples/web/content/apprtc/bigquery/analytics_schema.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "name": "impression_name",
```

Comment:
not sure how this might work so open to suggestions. we're not directly passing the values through to an instance of the schema because of how timestamps work so it's a bit harder to do reflection here. the strings are what identify the fields and the strings shouldn't change. that is, fields of a table cannot be removed without rebuilding the table. I think making them static members of the class would be OK. we could initialize the consts at runtime for the class as well.